federation/pkg/federation-controller/util/meta_test.go

```diff
@@ -115,3 +115,23 @@ func TestObjectMetaAndSpec(t *testing.T) {
 	assert.False(t, ObjectMetaAndSpecEquivalent(&s1, &s3))
 	assert.False(t, ObjectMetaAndSpecEquivalent(&s2, &s3))
 }
+
+func TestIsFederationOnlyObject(t *testing.T) {
```

Comment:
(No action required) Looks good, thank you. I'm posting a link to a comment I made in another review in case you're interested in refining future test efforts (specifically using a map instead of a slice to name the test and using subtests):

https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/44139#-KhOb8dQMVNcSYlPCe_2